tests/manager_tests/test_cve_handler.py

```diff
@@ -86,7 +86,7 @@ def test_patch_access(self, monkeypatch):
     def test_cve_has_rules(self):
         response = self.vfetch('cves/CVE-2017-8').check_response()
         assert len(response.body.data.attributes.rules) == 2
-        assert response.body.data.attributes.rules[0].associated_cves == ['CVE-2017-8', 'CVE-2018-1']
+        assert response.body.data.attributes.rules[0].associated_cves == ['CVE-2018-1', 'CVE-2017-8']
```

Comment:
could you please rewrite the `assert` to something like:
```
assert len(response.body.data.attributes.rules[0].associated_cves) == 2
assert 'CVE-2017-8' in response.body.data.attributes.rules[0].associated_cves
assert 'CVE-2018-1' in response.body.data.attributes.rules[0].associated_cves
```
This way we won't run into problems with order again